Errors of running UFS-SRW-AQM develop branch on Gaea platform

We are trying to get the AQM develop branch running on Gaea C5 and C6 using the current module settings and configuration files. The model can be built successfully (and slowly) with many warnings. We fixed some module and script issues when running the model, but we are currently stuck in the forecast job with an MPI error: "[CRAYBLAS_WARNING] Application linked against multiple cray-libsci libraries". Any help will be greatly appreciated.      

Additional questions. (1) We moved NEXUS emission files to C5 (/gpfs/f5/arl/world-shared/Emissions) and C6 (/gpfs/f6/bil-fire3/world-shared/Emissions). Can these locations be added to the default gaeac6.yaml file together with other fixed locations? (2) It seems HPSS can not be accessed through htar now. Will it be fixed in the future?

## Current behavior
Forecast job cannot run.

## Machines affected
Gaea C6 and C5

## Steps To Reproduce
1. Follow instructions to download and build the model with ATMAQ application. 
2. Modify "modulefiles/tasks/python_srw.lua" to use "srw_aqm" environment. In our test, this change does not require creating local "taskname.local.lua" files for each of the aqm_prep tasks, although we understand this is quite local and not flexible enough. But we think the "python_srw_aqm.lua"  can load the modules for all the aqm_prep tasks. It may not be necessary to create a "taskname.local.lua" file for each aqm_prep task.
3. Modify “ush/machine/gaeac6.yaml” to point to the emission files
4. Modify /scripts/exsrw_aqm_lbcs.sh and /scripts/exsrw_fire_emission.sh to fix some typos (can start a pull request about this if necessary).
5. Launch the jobs and see the MPI error in (/gpfs/f6/bil-fire3/scratch/Wei.K.Li/expt_dirs/aqm_AQMNA13km_warmstart_develop/log/run_fcst_mem000_2023111000.log)


## Additional Information (optional)
Any way we can use crontab on Gaea C6? I know scron is used there but cron is a lot easier.

@benkozi Thank you!

However, upon a quick test of latest SRW-App [develop] or in your branch[ [gaeac56_jobscripts]](https://github.com/benkozi/ufs-srweather-app/tree/feature/gaeac56_jobscripts) just now we are experiencing a new issue when building on C6:


```
... Generate CMAKE configuration ...
-- The C compiler identification is unknown
-- The CXX compiler identification is unknown
-- The Fortran compiler identification is unknown
Error:  Unable to find cray-libsci/25.03.0 libraries compatible with intel-classic/2023.2.0 targeting 'x86-genoa'.

-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - failed
-- Check for working C compiler: /opt/cray/pe/craype/2.7.30/bin/cc
-- Check for working C compiler: /opt/cray/pe/craype/2.7.30/bin/cc - broken
CMake Error at /autofs/ncrc-svm1_proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/unified-env/install/intel/2023.2.0/cmake-3.23.1-ixiyqlk/share/cmake-3.23/Modules/CMakeTestCCompiler.cmake:69 (message):
  The C compiler

    "/opt/cray/pe/craype/2.7.30/bin/cc"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /gpfs/f6/bil-fire3/proj-shared/Patrick.C.Campbell/run_ufs-srw-app_aqm/run_na_test_ufs-srw-app-develop_20250403/build/CMakeFiles/CMakeTmp

    Run Build Command(s):/usr/bin/gmake -f Makefile cmTC_c3ada/fast && /usr/bin/gmake  -f CMakeFiles/cmTC_c3ada.dir/build.make CMakeFiles/cmTC_c3ada.dir/build
    gmake[1]: Entering directory '/gpfs/f6/bil-fire3/proj-shared/Patrick.C.Campbell/run_ufs-srw-app_aqm/run_na_test_ufs-srw-app-develop_20250403/build/CMakeFiles/CMakeTmp'
    Building C object CMakeFiles/cmTC_c3ada.dir/testCCompiler.c.o
    /opt/cray/pe/craype/2.7.30/bin/cc   -diag-disable=10441  -o CMakeFiles/cmTC_c3ada.dir/testCCompiler.c.o -c /gpfs/f6/bil-fire3/proj-shared/Patrick.C.Campbell/run_ufs-srw-app_aqm/run_na_test_ufs-srw-app-develop_20250403/build/CMakeFiles/CMakeTmp/testCCompiler.c
    Error:  Unable to find cray-libsci/25.03.0 libraries compatible with intel-classic/2023.2.0 targeting 'x86-genoa'.

    gmake[1]: *** [CMakeFiles/cmTC_c3ada.dir/build.make:78: CMakeFiles/cmTC_c3ada.dir/testCCompiler.c.o] Error 1
    gmake[1]: Leaving directory '/gpfs/f6/bil-fire3/proj-shared/Patrick.C.Campbell/run_ufs-srw-app_aqm/run_na_test_ufs-srw-app-develop_20250403/build/CMakeFiles/CMakeTmp'
    gmake: *** [Makefile:127: cmTC_c3ada/fast] Error 2

```

Related to cray/intel libraries.  Do we need a new issue for this?

@angehung5

--- a/modulefiles/build_gaeac6_intel.lua
+++ b/modulefiles/build_gaeac6_intel.lua
@@ -23,8 +23,8 @@ load("srw_common")
 load(pathJoin("nco", os.getenv("nco_ver") or "5.0.6"))
 load(pathJoin("prod_util", os.getenv("prod_util_ver") or "2.1.1"))
 
-unload("darshan-runtime/3.4.4")
-unload("cray-pmi/6.1.13")
+unload("darshan-runtime")
+unload("cray-libsci")
 
 setenv("CFLAGS","-diag-disable=10441")
 setenv("FFLAGS","-diag-disable=10441")

